tests/ord-service/tests/api_test.go

```diff
@@ -819,7 +819,7 @@ func TestORDServiceSystemDiscoveryByApplicationTenantID(t *testing.T) {
 	certSubject := strings.Replace(conf.ExternalCertProviderConfig.TestExternalCertSubject, conf.ExternalCertProviderConfig.TestExternalCertCN, "ord-svc-system-discovery", -1)
 
 	// We need an externally issued cert with a subject that is not part of the access level mappings
-	externalCertProviderConfig := createExternalConfigProvider(certSubject)
+	externalCertProviderConfig := createExternalConfigProvider(certSubject, conf.CertSvcInstanceTestSecretName)
```

Comment:
I have some questions/concerns about those two tests - `TestORDServiceSystemDiscoveryByApplicationTenantID` and `TestORDServiceSystemDiscoveryByApplicationTenantIDUsingProviderCSM` but for now I'll approve the PR so we can proceed with the next steps. After that, we can follow up on that